rafter/step2.md

```diff
@@ -1,47 +1,53 @@
-**Use Case 1 - Rafter as a static sites host**
-
-This steps shows how you can easily create a Bucket and then push to it an Asset. The Asset here is a package containing all static files needed for a website, which are HTML, JS and CSS files.
-
-Set variable to point to a location of released website
-`export GH_WEBPAGE_URL=https://github.com/kyma-project/examples/archive/master.zip`{{execute}}
-
-Create a Bucket that will contain final website files
-```
-cat <<EOF | kubectl apply -f -
-apiVersion: rafter.kyma-project.io/v1beta1
-kind: Bucket
-metadata:
-  name: pages
-  namespace: default
-spec:
-  region: "us-east-1"
-  policy: readonly
-EOF
-```{{execute}}
-
-Create an Asset resource. Controller fetches the Asset from the location provuded in the `spec.source.url`. In this example you can see that what is fetched is a package from which only specific directory is filtered out.
-```
-cat <<EOF | kubectl apply -f -
-apiVersion: rafter.kyma-project.io/v1beta1
-kind: Asset
-metadata:
-  name: webpage
-  namespace: default
-spec:
-  source:
-    url: ${GH_WEBPAGE_URL}
-    mode: package
-    filter: /asset-store/webpage/
-  bucketRef:
-    name: pages
-EOF
-```{{execute}}
-
-Make sure that the Asset is in Ready status, as it means that fetching, unpacking and filtering is completed
-`kubectl get assets webpage -o jsonpath='{.status.phase}'`{{execute}}
-
-Add Bucket name to the environment variable
-`export BUCKET_NAME=$(kubectl get bucket pages -o jsonpath='{.status.remoteName}')`{{execute}}
-
-Echo and then copy the link and open it in a browser to see the website works fine
-`echo https://[[HOST_SUBDOMAIN]]-31311-[[KATACODA_HOST]].environments.katacoda.com/$BUCKET_NAME/webpage/examples-master/asset-store/webpage/index.html`{{execute}}
\ No newline at end of file
+**Use Case 1 - Rafter as static sites host**
+
+In this scenario, you will create a Bucket and then push an Asset to it. The Asset is a package containing all static files needed to build a website, such as HTML, JS and CSS files. Follow these steps:
+
+1. Export a GitHub webpage URL of ready-to-use website sources.
```

Comment:
```suggestion
1. Export variable with ready-to-use sources of a simple Website.
```

Please also apply this suggestion to other parts where you write about exporting variables, just write `Export variable`

Reply:
I will follow the "template" we usually use in Kyma:

```suggestion
1. Export the URL to ready-to-use sources of a simple website as an environment variable. Run:
```